pkg/event/event.go

```diff
@@ -373,13 +418,15 @@ connect:
 			}
 
 			// update clock class
-			go func() {
-				select {
-				case clockClassRequestCh <- ClockClassRequest{cfgName: event.CfgName, gmState: gmState, clockType: event.ClockType}:
-				default:
-					glog.Error("clock class request not read")
-				}
-			}()
+			if gmState != lastGmState {
```

Comment:
this will always be false due to  line 407 set lastGmState = gmState

Reply:
review again pls